Netatmo: re-add content from 40001 (#40003)
home-assistant/home-assistant.github.io: source/_integrations/netatmo.markdown

To edit an existing area, enter its name and follow the dialog.

> By default, the opening entity shows up as a generic binary sensor. You can manually change its *device class* (e.g. `door`, `window`) to better match the real appliance.

Author: ```suggestion
By default, the opening entity appears as a generic binary sensor. You can manually change its *device class* (for example, `door`, or `window`) to better match the real appliance.
```

## Binary sensor

> The `netatmo` binary sensor platform shows connectivity entities for the [Netatmo Smart Home Weather Station](https://www.netatmo.com/smart-weather-station).

Reviewer: ```suggestion
The binary sensor platform shows connectivity entities for the [Netatmo Smart Home Weather Station](https://www.netatmo.com/smart-weather-station).
```
Or maybe the binary sensor platform from the Netatmo integration, but I guess we can remove the `netatmo` wording now as that usually is bound to when you were using it in YAML
